scripts/update-index-state-hashes.nix

```diff
@@ -31,7 +31,8 @@ writeShellScriptBin "update-index-state-hashes" ''
 
       # ensure we don't generate the $start date twice (skip the first invocation).
       if [[ "''${dt}T00:00:00Z" != "$start" ]]; then
-        truncate-index -o ''${dt}-01-index.tar.gz -i ${fetchurl "https://hackage.haskell.org/01-index.tar.gz"} -s "''${dt}T00:00:00Z"
+        curl "https://hackage.haskell.org/01-index.tar.gz" --output index.tar.gz --fail-early
```

Comment:
Doesn't this require `curl` to be in path? And don't we need `cacert` as well?

Reply:
:man_facepalming:

Reply:
I added curl, and ran it in a pure nix shell and it works. I don't think we seem to need `cacert`.

Reply:
Awesome, thanks!